ui/shared/AdditionalInformationTable.jsx

```diff
@@ -0,0 +1,160 @@
+  {
+    name: 'Individual alert',
+    imageBeforeHover: (
+      <img src={individualAlert} alt="Individual alert before hover." />
+    ),
+    imageAfterHover: (
+      <img src={individualAlertHovered} alt="Individual alert after hover." />
+    ),
+  },
+  {
+    name: 'Individual alert - absolute difference',
+    imageBeforeHover: (
+      <img src={individualAlertDetails} alt="Individual alert details." />
+    ),
+    imageAfterHover: (
+      <img
+        src={absoluteDifference}
+        alt="Individual alert details - absolute difference."
+      />
+    ),
+  },
+  {
+    name: 'Individual alert - magnitude of change',
+    imageBeforeHover: (
+      <img src={individualAlertDetails} alt="Individual alert details." />
+    ),
+    imageAfterHover: (
+      <img
+        src={magnitudeOfChange}
+        alt="Absolute difference details - magnitude of change."
+      />
+    ),
+  },
+  {
+    name: 'Individual alert - confidence',
+    imageBeforeHover: (
+      <img src={individualAlertDetails} alt="Individual alert details." />
+    ),
+    imageAfterHover: (
+      <img src={confidence} alt="Absolute difference details - confidence." />
+    ),
+  },
+];
+
+const compareViewHoverableItems = [
+  {
+    name: 'Filter buttons - Hide uncomparable Results',
+    imageBeforeHover: (
+      <img src={compareViewFilters} alt="Filter buttons in Compare View." />
+    ),
+    imageAfterHover: (
+      <img
+        src={hideUncomparableResults}
+        alt="Hide uncomparable results filter in Compare View."
+      />
+    ),
+  },
+  {
+    name: 'Filter buttons - Show only important changes',
+    imageBeforeHover: (
+      <img src={compareViewFilters} alt="Filter buttons in Compare View" />
+    ),
+    imageAfterHover: (
+      <img
+        src={showOnlyImportantChanges}
+        alt="Show only important changes filter in Compare View"
+      />
+    ),
+  },
+  {
+    name: 'Filter buttons - Hide uncertain results',
+    imageBeforeHover: (
+      <img src={compareViewFilters} alt="Filter buttons in Compare View" />
+    ),
+    imageAfterHover: (
+      <img
+        src={hideUncertainResults}
+        alt="Hide uncertain results filter in Compare View"
+      />
+    ),
+  },
+  {
+    name: 'Filter buttons - Show only noise',
+    imageBeforeHover: (
+      <img src={compareViewFilters} alt="Filter buttons in Compare View" />
+    ),
+    imageAfterHover: (
+      <img src={showOnlyNoise} alt="Show only noise filter in Compare View" />
+    ),
+  },
+];
+
+const AdditionalInformationTable = function AdditionalInformationTable() {
+  return (
+    <div className="card">
+      <div className="card-header">
+        <h3>Elements with extra information</h3>
+      </div>
+
+      <div className="card-body panel-spacing">
+        <table>
+          <tbody>
+            <tr>
+              <th colSpan="1">Alerts View</th>
+              <th colSpan="1">Before hover</th>
+              <th colSpan="1">After hover</th>
```

Comment:
The user guide was initially intended for Treeherder-only. With this PR, there's nothing telling the user these new instructions are for Perfherder.

Maybe create a separate `Perfherder User Guide` group, similar to the original `Treeherder User Guide` one?